recipes/semver/all/conanfile.py

```diff
@@ -0,0 +1,67 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class SemverConan(ConanFile):
+    name = "semver"
+    description = "Semantic Versioning for modern C++"
+    topics = ("conan", "semver", "semantic", "versioning")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/Neargye/semver"
+    license = "MIT"
+    generators = "cmake", "cmake_find_package_multi"
+    settings = "os", "arch", "compiler", "build_type"
```

Comment:
Only compiler is used from what I can see

Reply:
Removed not used settings. Left  `compiler` and `build_type` because it wouldn't build without them:

> ERROR: 'settings.build_type' doesn't exist